antlr_tsql/tsql.g4

```diff
@@ -1172,6 +1200,11 @@ on_off
     | OFF
     ;
 
+time_io
+    : TIME
+    | IO
```

Comment:
What is this alternative based on?

Reply:
This is based on the four following queries:
SET STATISTICS TIME ON
SET STATISTICS TIME OFF
SET STATISTICS IO ON
SET STATISTICS IO OFF

Because ON/OFF was made as an alternative, I also made TIME/IO as one.

Reply:
Solved